x-pack/solutions/observability/plugins/synthetics/server/synthetics_service/private_location/package_policy_service.ts

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { NewPackagePolicyWithId } from '@kbn/fleet-plugin/server/services/package_policy';
+import type { SyntheticsServerSetup } from '../../types';
+
+export class PackagePolicyService {
+  private readonly server: SyntheticsServerSetup;
+
+  constructor(_server: SyntheticsServerSetup) {
```

Comment:
```suggestion
  constructor(_server: Dependencies) {
```
i wish we can do this as well, hopefully in a follow up, we call it what it actually is.